include/continuous_action_response.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <cstddef>
+#include <iterator>
+#include <vector>
+#include <string>
+
+namespace reinforcement_learning {
+  class api_status;
+
+  class continuous_action_response {
```

Comment:
Types and methods in the public API must be documented.

Reply:
I can add it but as an fyi `ranking_response.h` is the only response api that has documentation

Reply:
Done - documented public api methods in `continuous_action_response.h` (also cleaned up a bit the move/copy constructor/operators for that object).

Also added documentation for all of the new `request_continuous_action` methods in `live_model.h`. 

I can document the ones in `slates_response.h` and `decision_response.h` in a separate PR for consistency.

Reply:
I'm documenting slates as part of another PR, thanks for the offer though :)